bootstrap.js

```diff
@@ -257,6 +257,10 @@ let prefObserver = {
     case "theme":
     case "thumbs.contain":
     case "thumbs.hidebuttons":
+      enumerateTabs(function(aWindow) {
+        aWindow.newTabTools.updateUI();
+      });
+      break;
```

Comment:
@skywinds5012 Can you verify the patch, I think it will take effect for this issue.